### PR TITLE
first look into hangman

### PR DESCRIPTION
**Pull request by Henrynunez112**

### CODE REVIEW

**Comment by Henrynunez112** [comment]
I have officially submitted my work. But I will be debugging tonight... wish me luck!!

File: HumanPlayer.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class HumanPlayer{
+    constructor(name){
+        this.name = name;
+
+    }
+    getMove(){
+        let move = readline.question(`Hello ${this.name}, I am glad you decided to play, please pick a word... any word...`);
+        move.toLowerCase();
```
Comment (xpectro93) [inline HumanPlayer.js]: In here to are changing move but not redefining or updating it. You would want to have `move = move.toLowerCase()`

File: ComputerPlayer.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const allTheWords = require("./words.js")
+
+class ComputerPlayer{
+    constructor(){
+        this.computer = computer;
```
Comment (xpectro93) [inline ComputerPlayer.js]: typically, if you have `this.computer = computer` you want to have computer in your constructor, so you can set 
`this.computer` to whatever value you input on the constructor.

File: Game.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+const HumanPlayer = require("./HumanPlayer.js")
+const ComputerPlayer = require("./ComputerPlayer.js")
+const Board = require("./Board.js")
+const readline = require('readline-sync');
+class Game{
+    constructor(){
+        this.board = new Board(this.computer.secretWord());
+        this.player = player;
+        this.computer = new ComputerPlayer();
```
Comment (xpectro93) [inline Game.js]: You have to keep track of what your classes or functions take in when being invoked, this will help you a lot on the long run.

**Pull request closed without merging**